generatebundlefile/scripts/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -e
+set -o pipefail
+
+USR_BIN=/usr/bin
+
+if [ ! -d "/root/.docker" ]; then
+    mkdir -p /root/.docker
+fi
```

Comment:
As I recall, the build tooling does something similar to this

Reply:
Yea, this is a separate build job, so it's not using build tooling repo. This stuff and the `docker-ecr-config` is used to provide the Docker information about which ECR repos it can communicate with. We need this since we're doing docker and Oras push/pull in this step.